tests/framework/clients/block.go

```diff
@@ -48,9 +48,10 @@ func CreateBlockOperation(k8shelp *utils.K8sHelper, manifests installer.CephMani
 
 // BlockCreate Function to create a Block using Rook
 // Input parameters -
-// manifest - pod definition that creates a pvc in k8s - yaml should describe name and size of pvc being created
-// size - not user for k8s implementation since its described on the pvc yaml definition
-// Output - k8s create pvc operation output and/or error
+// manifest - pod definition that creates a PVC in k8s - YAML should describe the name and size of the PVC being created
+// size - not user for k8s implementation since it is descriebd in the PVC YAML definition
+// Output - k8s create PVC operation output and/or error
+
```

Comment:
Revert this file? Or why the blank line?

Reply:
The only change should have been capitalization.